### ISSUE TITLE
Equivalent when I don't know the model/serializer at play?

### ISSUE THREAD

**Issue by mjobin-mdsol** [okuramasafumi/alba#131]
I am looking for an AMS equivalent to 

```ruby
          instance_serialization = ActiveModelSerializers::SerializableResource.new(local_instance).as_json
```
basically, I don't know what object `local_instance` is and what serializer class I should be using. AMS does that for me automatically.

Thank you

**Comment by okuramasafumi** [comment]
Hi @mjobin-mdsol I created a PR at #132 which I believe solves your problem. Please take a look at it :)

**Comment by okuramasafumi** [comment]
Hi @mjobin-mdsol I merged #132 to resolve this issue. I like the idea.
